src/spec-node/devContainersSpecCLI.ts

```diff
@@ -90,7 +90,7 @@ function provisionOptions(y: Argv) {
 		'docker-compose-path': { type: 'string', description: 'Docker Compose CLI path.' },
 		'container-data-folder': { type: 'string', description: 'Container data folder where user data inside the container will be stored.' },
 		'container-system-data-folder': { type: 'string', description: 'Container system data folder where system data inside the container will be stored.' },
-		'workspace-folder': { type: 'string', description: 'Workspace folder path. The devcontainer.json will be looked up relative to this path.' },
+		'workspace-folder': { type: 'string', default: '.', description: 'Workspace folder path. The devcontainer.json will be looked up relative to this path.' },
```

Comment:
We should treat it the same with the other commands. Would it make sense to search for the first parent folder with a `.devcontainer` folder (or a `.devcontainer.json` file)? Similar to what `git` does.

Reply:
It would make perfect sense, but this is above my TypeScript knowledge to do that.
Feel free to close this PR to let a futur contributor to may be implement this.

Reply:
Sounds good, closing for now, thanks!